src/components/__tests__/FormLogin.cy.js

```diff
@@ -22,19 +22,30 @@ const contactEmail = rideToWorkByBikeConfig.contactEmail;
 const classSelectorQNotificationMessage = '.q-notification__message';
 
 // variables
+const { apiBase, urlApiLogin, urlApiRefresh } = rideToWorkByBikeConfig;
+const apiLoginUrl = `${apiBase}${urlApiLogin}`;
+const apiRefreshUrl = `${apiBase}${urlApiRefresh}`;
+
 const username = 'test@example.com';
 const password = 'example123';
-const tokenAccess = '1234567890';
-const tokenRefresh = '0987654321';
+
+const tokenAccess =
```

Comment:
token* var should be as Cy fixture.

Reply:
Refactored into a fixture.
Also updated the time variables in test for better readability.